dipy/workflows/segment.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import division, print_function, absolute_import
+
+from glob import glob
+from os.path import join, basename, splitext, dirname, isabs, exists
+from os import makedirs
+
+import nibabel as nib
+import numpy as np
+
+from dipy.segment.mask import median_otsu
+
+
+def median_otsu_bet(input_file, out_dir, save_masked=False, median_radius=4,
```

Comment:
I am suggesting that all the functions in workflows should end with the word ``flow``.
``def median_otsu_flow``
and that all parameters should be strings by default. In that way there will be no need for parsing the arguments outside of the flow. That would make life easier when integrating with nipype and client/server web interfaces.
Look for inspiration here:
https://github.com/Garyfallidis/dipy/blob/slr_tutorials/dipy/workflows/segment.py#L54